gitops/overlays/int1/configs/frontend/config.conf

```diff
@@ -1,11 +1,13 @@
 # LOG_LEVEL=debug
 
-AUTH_LOGOUT_REDIRECT_URL=https://srv113-i.sade.hrdc-drhc.gc.ca/ecas-seca/raoidc/v1/logout?client_id={clientId}&shared_session_id={sharedSessionId}&ui_locales={uiLocales}
-AUTH_RAOIDC_BASE_URL=https://srv113-i.sade.hrdc-drhc.gc.ca/ecas-seca/raoidc/v1
+AUTH_LOGOUT_REDIRECT_URL=https://cdcp-int1.dev-dp.dts-stn.com/
+AUTH_RAOIDC_BASE_URL=https://cdcp-int1.dev-dp-internal.dts-stn.com/oidc
 AUTH_RAOIDC_CLIENT_ID=CDCP
-AUTH_RASCL_LOGOUT_URL=https://srv113-i.sade.hrdc-drhc.gc.ca/ecas-seca/rascl/Support/GlobalLogout.aspx
+AUTH_RASCL_LOGOUT_URL=https://cdcp-int1.dev-dp.dts-stn.com/
+MOCK_AUTH_ALLOWED_REDIRECTS=https://cdcp-int1.dev-dp.dts-stn.com/auth/callback/raoidc
```

Comment:
This might be a nitpick.. but hear me out.

Moving this down below `ENABLED_MOCKS` will make it easier to diff this file against our `.env.example` file to see what is (or isn't) missing.

When the envars are not in the same order, diff'ing becomes practically useless, which means you have to read through the full `.env.example` file line by line to make sure you have all the required settings.

Reply:
I recently reordered all of these .conf files for this exact reason.